shortlister/controller.py

```diff
@@ -324,13 +349,23 @@ def clear_filter(self, k=None):
         self.ctx.applicants = self.shortlist.applicants
         self.show_applicants_table()
 
+    def export_applicants_excel(self, k=None):
+        """Export selected applicants to Excel spreadsheet"""
+
+        filename = Path(input("Save as (filename):") + ".xlsx")
```

Comment:
Create a default filename, so if the user presses return (without providing any input), you'll get a meaningful filename. Using the timestamp you could have something like "<role-id>_<timestamp>.xlsx" e.g.`test-role_202501130948.xlsx`